### PR TITLE
Update template docs

### PR DESCRIPTION
**Pull request by davidstauffer**
Closes

**What changed?**

**Why was this change made?**

**How was this change implemented?**

**How did you validate the change?**

**Release notes**

**Documentation Changes**

### CODE REVIEW

File: website/docs/gitops-templates/templates.mdx [hunk 1]
```diff
@@ -240,10 +240,30 @@ spec:
...
 You can provide additional metadata about the parameters to the templates in the `spec.params` section.
 
+### Required Parameters
+Currently templates use the `CLUSTER_NAME` to determine the path in gitrepo and `RESOURCE_NAME` for the file name when we create the PR and write to your git repository. 
+
+You must provide either:
+- `CLUSTER_NAME`
+- `RESOURCE_NAME`
+
+**Default path for templates**
```
Comment (foot) [inline website/docs/gitops-templates/templates.mdx:256]:
```suggestion
**Default path for templates**

```

File: website/docs/gitops-templates/templates.mdx [hunk 2]
```diff
@@ -240,10 +240,30 @@ spec:
...
 ```
 
-### Parameter metadata - `spec.params`
+
+## Parameters
 
 You can provide additional metadata about the parameters to the templates in the `spec.params` section.
 
+### Required Parameters
+Currently templates use the `CLUSTER_NAME` to determine the path in gitrepo and `RESOURCE_NAME` for the file name when we create the PR and write to your git repository.
```
Comment (sympatheticmoose) [inline website/docs/gitops-templates/templates.mdx:249]: - Would remove "Currently" - not required in technical docs
- Would be explicit - in the git repository, PR -> pull request
- Could optionally omit "and write to your git repository"

File: website/docs/gitops-templates/templates.mdx [hunk 3]
```diff
@@ -240,10 +240,30 @@ spec:
...
 
+### Required Parameters
+Currently templates use the `CLUSTER_NAME` to determine the path in gitrepo and `RESOURCE_NAME` for the file name when we create the PR and write to your git repository. 
+
+You must provide either:
+- `CLUSTER_NAME`
+- `RESOURCE_NAME`
+
+**Default path for templates**
+The default path for a template has a few components:
```
Comment (foot) [inline website/docs/gitops-templates/templates.mdx:258]:
```suggestion
The default path for a template has a few components:

```

File: website/docs/gitops-templates/templates.mdx [hunk 4]
```diff
@@ -240,10 +240,30 @@ spec:
...
-### Parameter metadata - `spec.params`
+
+## Parameters
 
 You can provide additional metadata about the parameters to the templates in the `spec.params` section.
 
+### Required Parameters
+Currently templates use the `CLUSTER_NAME` to determine the path in gitrepo and `RESOURCE_NAME` for the file name when we create the PR and write to your git repository. 
+
+You must provide either:
```
Comment (sympatheticmoose) [inline website/docs/gitops-templates/templates.mdx:251]: implies mutually exclusive, is this the case?

File: website/docs/gitops-templates/templates.mdx [hunk 5]
```diff
@@ -240,10 +240,30 @@ spec:
...
+- `CLUSTER_NAME`
+- `RESOURCE_NAME`
+
+**Default path for templates**
+The default path for a template has a few components:
+- From the params: `CLUSTER_NAME` or `RESOURCE_NAME`, **required**.
+- From the params: `NAMESPACE`, default: `default`
+- From values.yaml: `values.config.capi.repositoryPath`, default: `./clusters/management/clusters`
```
Comment (sympatheticmoose) [inline website/docs/gitops-templates/templates.mdx:259]: maybe worth calling out that this is the Weave GitOps Enterprise helm chart?

**Review by sympatheticmoose — APPROVED** [review]
Nothing blocking - just suggestions

**Review by foot — APPROVED** [review]

File: website/docs/gitops-templates/templates.mdx (hunk 4 above)
Comment (davidstauffer) [reply website/docs/gitops-templates/templates.mdx:252]: not exclusive. Both are used with a slightly different purpose.

File: website/docs/gitops-templates/templates.mdx (hunk 2 above)
Comment (davidstauffer) [reply website/docs/gitops-templates/templates.mdx]: Done. Thanks.

File: website/docs/gitops-templates/templates.mdx (hunk 4 above)
Comment (davidstauffer) [reply website/docs/gitops-templates/templates.mdx]: rephrased, changes soon will make this much simpler and more explicit. (path feature)

**Pull request merged**